apps/experiments/views/experiment_routes.py

```diff
@@ -42,6 +42,12 @@ def get_success_url(self):
     def form_valid(self, form):
         form.instance.team = self.request.team
         self.object = form.save(commit=False)
+
+        # Set parent_version and child_version
+        parent_experiment = get_object_or_404(Experiment, id=self.kwargs["experiment_id"])
+        self.object.parent_version = parent_experiment.version_number
+        self.object.child_version = Experiment.objects.get(id=self.object.child_id).version_number
```

Comment:
This is something to talk about. Should the 'core' route config always point to the 'core' version of the linked bots?

When you create a new version of the route bot you could have options to use the 'live' version of the linked bots or to select a specific version for each?

Reply:
to clarify, is this the options you are thinking for what the routes are referencing for the experiment (I'll use "working" to describe the draft exp state):
1. parent: working -> child: working
2. parent: working -> child: versioned
3. parent: versioned -> child: versioned

right now are just options 1 and 3, but option 2 also makes sense to me to add

Reply:
1 & 3 is great

Reply:
ah gotcha yes that was an oversight: https://github.com/dimagi/open-chat-studio/pull/604#:~:text=%E2%80%A6-,3b3c977,-when%20creating%20a

mods will still need to be changed here